redux/Books/actions.js

```diff
@@ -6,18 +6,22 @@ export const actionTypes = {
   GET_BOOKS_FAILURE: 'GET_BOOKS_FAILURE'
 };
 
-const getAllBooks = () => dispatch => {
-  dispatch({ type: actionTypes.GET_BOOKS });
-  BookService.getBooks().then(res => {
-    if (!res.ok) {
-      return dispatch({ type: actionTypes.GET_BOOKS_FAILURE, res });
-    }
-    return dispatch({ type: actionTypes.GET_BOOKS_SUCCESS, res });
-  });
+const privateBookActions = {
+  getBooksSuccess: books => ({ type: actionTypes.GET_BOOKS_SUCCESS, payload: books }),
+  getBooksFailure: error => ({ type: actionTypes.GET_BOOKS_FAILURE, payload: error })
 };
 
-const actionCreators = {
-  getAllBooks
+export const bookActions = {
+  getBooks: () => async dispatch => {
+    dispatch({ type: actionTypes.GET_BOOKS });
+    // import BookService from your service file
```

Comment:
why do you leave this comment?

Reply:
copy paste alert :laughing: 

I extracted action and reducer from Trello card documentation
https://trello.com/c/FYC7G0J0/22-integracion-con-books